juriscraper/pacer/appellate_attachment_page.py

```diff
@@ -0,0 +1,211 @@
+from __future__ import print_function
+
+import pprint
+import re
+import sys
+
+from lxml import html
+
+from juriscraper.lib.html_utils import strip_bad_html_tags_insecure
+from juriscraper.lib.string_utils import force_unicode
+from .reports import BaseReport
+from ..lib.log_tools import make_default_logger
+
+logger = make_default_logger()
+
+
+class AppellateAttachmentPage(BaseReport):
+    """An object for querying and parsing the appellate att. page report."""
+
+    PATH = "doc1/"
+
+    def __init__(self, court_id, pacer_session=None):
+        super(AppellateAttachmentPage, self).__init__(court_id, pacer_session)
+        # Note that parsing appellate attachment pages does not reveal the
+        # document number, only the attachment numbers.
+
+    def query(self, document_number):
+        """Query the "attachment page" endpoint and set the results to self.response.
+
+        :param document_number: The internal PACER document ID for the item.
+        :return: a request response object
+        """
+        assert (
+            self.session is not None
+        ), "session attribute of DocketReport cannot be None."
+        # coerce the fourth digit of the document number to 1 to ensure we get
+        # the attachment page.
+        document_number = document_number[:3] + "0" + document_number[4:]
```

Comment:
This comment and the code that follow it are at odds?

Reply:
This was meant to be consistent with the attachment_page I modeled this page on.

Reply:
Will update it.